pkg/apis/pipeline/v1/matrix_types.go

```diff
@@ -110,7 +110,7 @@ func (m *Matrix) CountCombinations() int {
 // countGeneratedCombinationsFromParams returns the count of combinations of Parameters generated from the matrix
 // parameters
 func (m *Matrix) countGeneratedCombinationsFromParams() int {
-	if !m.hasParams() {
+	if !m.HasParams() {
```

Comment:
Thanks for the changes @EmmaMunley , can we elaborate the reasons why we made those functions exported? and if so might as well update it in the PR message? 🙏